src/Raven.Server/Documents/Indexes/Persistence/Lucene/LuceneAnalyzerAdapterForWriter.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.IO;
+using System.Text;
+using Corax.Pipeline;
+using Lucene.Net.Analysis;
+using Lucene.Net.Analysis.Tokenattributes;
+using LuceneAnalyzer = Lucene.Net.Analysis.Analyzer;
+using Raven.Server.Json;
+using Analyzer = Corax.Analyzers.Analyzer;
+using Token = Corax.Pipeline.Token;
+
+namespace Raven.Server.Documents.Indexes.Persistence.Lucene;
+
+internal sealed unsafe class LuceneAnalyzerAdapterForWriter : LuceneAnalyzerAdapter
```

Comment:
Maybe the following renames:
- `LuceneAnalyzerAdapterForQuerying` -> `LuceneQueryingAnalyzerAdapter`
- `LuceneAnalyzerAdapterForWriter` -> `LuceneIndexingAnalyzerAdapter`
?